shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala

```diff
@@ -1207,6 +1207,13 @@ object GlutenConfig {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("1GB")
 
+  val COLUMNAR_VELOX_MEM_INIT_CAPACITY =
+    buildConf("spark.gluten.sql.columnar.backend.velox.memInitCapacity")
+      .internal()
+      .doc("The initial memory capacity.")
```

Comment:
Could rephase to “The initial memory capacity to reserve for a newly created Velox query memory pool."

Reference https://github.com/facebookincubator/velox/blob/9c9712eac7e335b3636f5193b0a061c6015f9d10/velox/common/memory/Memory.h#L162